src/components/dao/LeaderboardTableRow.jsx

```diff
@@ -1,41 +1,37 @@
 import { Box, TableCell, TableRow, Text } from "@unioncredit/ui";
 import { Avatar, PrimaryLabel } from "components/shared";
 import { truncateAddress } from "utils/truncateAddress";
-import { ZERO } from "constants";
+import { ZERO, EIP3770 } from "constants";
 import { compactFormattedNumber } from "utils/format";
+import { Link } from "react-router-dom";
+import { useNetwork } from "wagmi";
 
 export function LeaderboardTableRow({ data }) {
-  const {
-    address,
-    unionBalance = ZERO,
-    votes = ZERO,
-    delegatedVotes = ZERO,
-    voteCount = 0,
-  } = data;
-
+  const { address, unionBalance = ZERO, votes = ZERO, delegatedVotes = ZERO, voteCount = 0 } = data;
+  const { chain } = useNetwork();
   return (
     <TableRow key={address}>
       <TableCell fixedSize>
-        <Avatar size={24} address={address} />
+        <Link to={`/profile/${EIP3770[chain.id]}:${data.address}`}>
```

Comment:
I think this line is causing the error when the user is not connected to a network. It is attempting to access `chain.id` for a chain that is undefined. The same thing is happening on line 20.

Suggested fix is probably to default to the `mainnet.id` if `chain` is not connected.